src/test/java/spoon/test/variable/VariableTest.java

```diff
@@ -191,4 +200,27 @@ public void testVarInLambda() {
         assertEquals("java.lang.Long", lambda.getParameters().get(1).getType().getQualifiedName());
         assertEquals("(var x,var y) -> x + y", lambda.toString()); // we should print var, if it was in the original code
     }
+
+    @ModelTest(value = "./src/test/resources/spoon/test/unnamed/UnnamedVar.java", complianceLevel = 22)
+    void testUnnamedVariable(Factory factory) throws IOException {
+        CtType<?> type = factory.Type().get("spoon.test.unnamed.UnnamedVar");
+        assertThat(type.getMethods()).isNotEmpty();
+        List<String> lines = java.nio.file.Files.readAllLines(type.getPosition().getFile().toPath());
```

Comment:
Do you use the FQN here by accident or necessity?

Reply:
`com.google.common.io.Files` is used in this class as well.